src/libcommon/utility/types.cpp

```diff
@@ -0,0 +1,730 @@
+/*
+ * Infomaniak kDrive - Desktop
+ * Copyright (C) 2023-2024 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <string>
+#include "types.h"
+#include <Poco/UnicodeConverter.h>
+
+namespace KDC {
+namespace typesUtility {
+std::wstring stringToWideString(const std::string &str) {
+    std::wstring wstr;
+    Poco::UnicodeConverter::convert(str, wstr);
+    return wstr;
+}
+}  // namespace typesUtility
+
+std::string enumClassToString(ReplicaSide e) {
```

Comment:
As discussed this morning, it would be great to call those helper functions only `toString`, `toWideString`, ...
And start using this rule everywhere

Reply:
I updated theses functions names here: [6d13250](https://github.com/Infomaniak/desktop-kDrive/pull/263/commits/6d1325018b6096378d1e90df97a0430510b30d4c)